src/components/Datefield/Datefield.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from "react";
+import { Icon } from "../Icon/Icon";
+import Calendar from "react-calendar";
+
+type IntrinsicElements = JSX.IntrinsicElements["input"];
+export interface Props extends IntrinsicElements {
+  id: string;
+  fieldName: string;
+  label: string;
+  hint?: boolean;
+  disabled?: boolean;
+  defaultDate?: string;
+  minDate?: string;
+  maxDate?: string;
+  value?: string;
+}
+
+/**
+ * Datefield Component
+ * @param {string}  id                     A unique identifier for the input.
+ * @param {string}  fieldName              Name of the input field.
+ * @param {string}  label                  Field label.
+ * @param {boolean} [hint]                 Boolean that shows or hide the date format hint, in the format mm/dd/yyyy.
+ * @param {string}  [disabled]             Controls whether or not the date picker is disabled to the user.
+ * @param {string}  [defaultDate]          The date picker input will set this value if it is a valid date. The date should be in the format mm/dd/yyyy
+ * @param {string}  [minDate]              The date picker will not allow a date selection before this date. The date should be in the format mm/dd/yyyy
+ * @param {string}  [maxDate]              The date picker will not allow a date selection after this date. The date should be in the format mm/dd/yyyy.
+ * @param {string}  [value]                Value of the input element.
+ */
+
+export const Datefield: React.FC<Props> = ({
+  id,
+  fieldName,
+  label,
+  value,
+  minDate,
+  maxDate,
+  defaultDate,
```

Comment:
![Screen Shot 2022-08-30 at 10 02 50 AM](https://user-images.githubusercontent.com/22647707/187457786-e10f9234-5564-4318-b577-d5e3ebe2c6db.png)
defaultDate seems to be displaying milliseconds